src/main/java/de/hysky/skyblocker/skyblock/waypoint/Waypoints.java

```diff
@@ -202,4 +236,26 @@ private static void render(WorldRenderContext context) {
             }
         }
     }
+
+	private enum OrderedAction implements StringIdentifiable {
+		NEXT,
+		PREVIOUS,
+		FIRST,
+		RESET;
+
+		private static final Codec<OrderedAction> CODEC = StringIdentifiable.createCodec(OrderedAction::values);
+		private static final ArgumentType ARGUMENT_TYPE = new ArgumentType();
```

Comment:
Not really standard to have this in a field, so I'm not sure if this would work correctly. You can see `CategoryArgumentType` and `DirectionArgumentType` for examples.